packages/config/presets.d.ts

```diff
@@ -1 +1 @@
-export * from "./lib/networks";
+export * from "./lib/presets";
```

Comment:
It took me a day to figure out why `mainnet` works but `minimal` didn't ;)

Reply:
Sorry :pray: